src/pages/ValidateLoginNewWorkspacePage.js

```diff
@@ -39,7 +43,11 @@ class ValidateLoginNewWorkspacePage extends Component {
             // by calling dismissModal(), the /v/... route is removed from history so the user will get taken to `/`
             // if they cancel out of the new workspace modal.
             Navigation.dismissModal();
-            Navigation.navigate(ROUTES.WORKSPACE_NEW);
+            if (this.props.betas) {
```

Comment:
Similar to comment in other file - how do you feel about using `lodashGet` here (like below)?

Reply:
No reason to use lodashGet here imo since the proptype and default are provided. The other example of lodashGet makes sense because accountID and validateCode are not defined in the defaultParams.

Reply:
True, but I did mention how `session.authToken` and `session.accountID` are in `defaultProps` **and** defaulted via `lodashGet` in the other component. But still I don't mind not using `lodashGet` here since `this.props.betas` is not nested in another onyx key 👍